packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -17,7 +17,7 @@ import { getStatusTranslator } from '../../utils';
 import { Toolbar } from './Toolbar';
 import { AppBarButtons } from './AppBarButtons';
 import { getOutboundShipmentListViewApi } from './api';
-import { pricingGuard, useOutboundShipmentApi } from '../api';
+import { useOutboundShipmentApi } from '../api';
```

Comment:
aaand from here down is the same thing: Remove handling of unions for nested fields, remove handling of `ConnectorError`